slog/slog.go

```diff
@@ -95,11 +95,16 @@ func Configure(cfg Config) error {
 		th.ReplaceAttr = func(groups []string, a slog.Attr) slog.Attr {
 			// Customize the name of some fields to match Google Cloud expectations
 			// More: https://cloud.google.com/logging/docs/agent/logging/configuration#process-payload
-			if a.Key == slog.LevelKey {
-				a.Key = "severity"
+			if len(groups) > 0 {
+				return a
 			}
-			if a.Key == slog.MessageKey {
+			switch a.Key {
+			case slog.LevelKey:
+				a.Key = "severity"
+			case slog.MessageKey:
 				a.Key = "message"
+			case "http_request":
+				a.Key, a.Value = convertHTTPRequest(a.Key, a.Value)
```

Comment:
not a huge deal..just thinking why the API is not just transforming the attribute, so it would be:

```
a = convertHTTPRequest(a)
```

Kinda like the signature of the ReplaceAttr function, it gets a slot.Attr and returns a transformed slog.Attr. But definitely not a big deal..or even a small deal I suppose...just wondering when I was re-reading the code :laughing: